MigrationTutorial/Models/V2/Customer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using MongoDB.Bson;
+using Realms;
+
+namespace MigrationTutorial.Models.V2
+{
+    public class Customer : RealmObject
+    {
+        [PrimaryKey]
+        public ObjectId Id { get; private set; } = ObjectId.GenerateNewId();
+
+        [Required]
+        public string? Name { get; set; }
+
+        public string Location { get; set; }
+
+        public Customer() { }
```

Comment:
Why the constructor?

Reply:
I left default public constructors everywhere. If stylistically it is considered bad practice, I'll remove them all. Let me know.

Reply:
I don't think it's a "bad practice", but it's not necessary, as it is implicit